src/schematron/schut-to-xspec.xsl

```diff
@@ -232,6 +234,26 @@
 
     <xsl:template match="@count | @label | @pending" as="empty-sequence()" mode="make-predicate" />
 
+    <xsl:template match="x:expect-assert | x:expect-report" as="text()" mode="make-text-predicate">
+        <xsl:variable name="x-expect-text-space-normalized" as="xs:string"
+            select="normalize-space(.)"/>
+        <xsl:variable name="apos-char" as="xs:string">'</xsl:variable>
+        <xsl:variable name="x-expect-text-apos-adjusted" as="xs:string"
+            select="replace($x-expect-text-space-normalized, $apos-char, $apos-char || $apos-char)"/>
+        <xsl:variable name="x-expect-text-content-wrapped" as="xs:string"
+            select="$apos-char || $x-expect-text-apos-adjusted || $apos-char"/>
```

Comment:
Looks like these lines can be replaced by a call to `x:quote-with-apos()` defined in `src/common/common-utils.xsl`, or at least `$apos-char` can be replaced by `$x:apos`.

Reply:
Great idea. I improved this piece of code in 551d3a63c74e23c02916db9455fb0c3bf0799d70.